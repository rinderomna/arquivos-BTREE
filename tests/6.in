6 tipo2 binario6.bin indice6.bin 3
1 id 797
2 id 890 ano 2012
1 ano 1991
