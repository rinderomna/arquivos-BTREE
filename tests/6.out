24014.030000
13141.180000
